frontend/src/components/TheLeafletMap.vue

```diff
@@ -111,13 +115,15 @@ onMounted(() => {
     }
   })
   const position = new Position()
-  leaflet.addControl(position)
-
+  //leaflet.addControl(position)
+ 
```

Comment:
remove commented code

Reply:
actually, commenting/removing this will create issues
![image](https://github.com/user-attachments/assets/2f3e051b-eaa0-4f5c-8839-99d7cf7f684b)
```
Uncaught TypeError: this._latlng is undefined
    updateHTML https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    setup https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    fire https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    _fireDOMEvent https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    _fireEvent https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    _handleMouseHover https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    _onMouseMove https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    oe https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    Fr https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    ln https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    _initContainer https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    onAdd https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    onAdd https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    _layerAdd https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    whenReady https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    addLayer https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    getRenderer https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    beforeAdd https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    addLayer https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    createLayers https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    _addFeatures https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    _requestFeatures https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    le https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    _requestFeatures https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    run https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    _createServiceCallback https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    onreadystatechange https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    NF https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    tte https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    _request https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    request https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    request https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    run https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    _requestFeatures https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    createCell https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    _createCell https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    _update https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    _setView https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    _resetView https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:101
    fire https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    _onZoomTransitionEnd https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    _catchTransitionEnd https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
    oe https://swotviz-dev.cuahsi.io/assets/index-5943de42.js:97
index-5943de42.js:101:91941
```